moveit_planners/chomp/chomp_interface/src/chomp_planning_context.cpp

```diff
@@ -34,18 +34,21 @@
 
 /* Author: Chittaranjan Srinivas Swaminathan */
 
-#include <memory>
 #include <chomp_interface/chomp_planning_context.h>
-#include <moveit/trajectory_processing/iterative_time_parameterization.h>
 #include <moveit/robot_state/conversions.h>
+#include <moveit/trajectory_processing/iterative_time_parameterization.h>
+
+#include "rclcpp/rclcpp.hpp"
 
 namespace chomp_interface
 {
+rclcpp::Logger LOGGER3 = rclcpp::get_logger("chomp_optimizer");
```

Comment:
Can you make the LOGGER's `static const`? This should allow you to drop the numbers. Alternative to `static` you can also wrap the loggers in anonymous namespaces

Reply:
Done.